winpr/tools/makecert/makecert.c

```diff
@@ -306,16 +306,19 @@ char* x509_name_parse(char* name, char* txt, int* length)
 
 char* x509_get_default_name()
 {
-	DWORD nSize = 0;
-	char* ComputerName;
+	TCHAR computerName[MAX_COMPUTERNAME_LENGTH + 1];
```

Comment:
use CHAR instead (since GetComputerNameExA and other ascii function versions are used below)